python/tchannel/tornado/dispatch.py

```diff
@@ -184,34 +175,33 @@ def close_argstreams(self, force=False):
             if stream.auto_close or force:
                 stream.close()
 
-    def arg1(self):
-        """get value for arg1
+    def get_header(self):
+        """Get the header value from the request.
 
-        :return: return the future object contains the value for arg1
+        :return: the value of header
```

Comment:
Actually these methods return futures. In the `:return:` section, mention that it returns "a future producing the value of [..]"